### PR TITLE
Mk evidences

### PR DESCRIPTION
**Pull request by mkarmona**

### CODE REVIEW

File: src/main/scala/io/opentargets/etl/backend/Evidence.scala [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.spark.sql._
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.types._
+import spark.{Helpers => H}
+
+object Evidence extends LazyLogging {
+  val normaliseString: String => String = c => c.replaceAll("[ .-]", "_")
+  val toCamelCase: String => String = c => {
```
Comment (JarrodBaker) [inline src/main/scala/io/opentargets/etl/backend/Evidence.scala:11]: I think Cinzia wrote a function in Helpers that does the same thing but on DataFrames. I think this method and `normaliseString` would be better there as we'll probably need them in several steps.

**Review by JarrodBaker — APPROVED** [review]

File: src/main/scala/io/opentargets/etl/backend/Evidence.scala (hunk 1 above)
Comment (cmalangone) [reply src/main/scala/io/opentargets/etl/backend/Evidence.scala:11]: Miguel and I already talked about this piece of code.
This is a manipulation of the current evidences index. 
The next step is to rewrite this step. So I am happy to refactor it later.

**Review by cmalangone — APPROVED** [review]

File: src/main/scala/io/opentargets/etl/backend/Evidence.scala (hunk 1 above)
Comment (mkarmona) [reply src/main/scala/io/opentargets/etl/backend/Evidence.scala:11]: @JarrodBaker @cmalangone we should try to decompose functions into simpler ones when it is possible and trying those to be composable in a way

**Review by JarrodBaker — APPROVED** [review]

**Comment by mkarmona** [comment]
@cmalangone @JarrodBaker My branch is too noisy and polluted so I guess squash and merge when reviewed would be the best.

**Pull request merged**